### PR TITLE
Add support for discarding collections.

### LINKED ISSUE #86: Add ability to discard drafts for collections

### PR DESCRIPTION
**Pull request by justinlittman**
closes #86

https://github.com/user-attachments/assets/c1ee43cc-86c2-4a89-be26-ad02ee8e4639

### CODE REVIEW

File: app/components/show/header_component.html.erb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<div class="d-flex justify-content-between">
```
Comment (justinlittman) [inline app/components/show/header_component.html.erb:1]: Extracted show page header into component.

File: config/locales/en.yml [hunk 2]
```diff
@@ -224,8 +224,9 @@ en:
             User agrees that, where applicable, content will not be used to identify or to otherwise infringe the privacy or
             confidentiality rights of individuals. Content distributed via the Stanford Digital Repository may be subject to
             additional license and use restrictions applied by the depositor.
-      errors:
+      messages:
```
Comment (justinlittman) [inline config/locales/en.yml:227]: Renamed to be consistent with works. (The messages are not all errors.)

File: app/components/show/header_component.html.erb (hunk 1 above)
Comment (mjgiarlo) [reply app/components/show/header_component.html.erb:1]: @justinlittman Is the idea that this header component will be used for both work show and collection show?

**Review by mjgiarlo — APPROVED** [review]

File: app/components/show/header_component.html.erb (hunk 1 above)
Comment (justinlittman) [reply app/components/show/header_component.html.erb:1]: Yes, I plugged it into both. (I decided to extract after realizing I was copying from work show to collection show ...)

**Pull request merged**